src/west/manifest.py

```diff
@@ -183,6 +189,18 @@ def _load(self, data, sections):
 
         manifest = data.get('manifest')
 
+        url = config.get('self', 'remote', fallback='origin')
```

Comment:
This fallback value doesn't look right; are you sure?

Reply:
No. and it points back to same problem as in: https://github.com/zephyrproject-rtos/west/pull/150/commits/792e7ea1fe4e527158d821470ddd1d712a4c5687#r247477779

What should we do if values are missing ?
In certain cases, we could read a remote from the git repo, if we know the path to it (also part of confiog file).
But then again, if there are multiple remotes defined (add by user), then which to use.

Reply:
As discussed elsewhere, solution for now is to remove the defaults and no specific error handling (meaning default python stack trace if key is missing)

Reply:
As discussed below I think a call to `log.die()` somewhere is appropriate.

Though upon reflection I don't think anything in `manifest.py` should call this function. I think calling `log.die` should be done by the routines under `src/west/commands` rather than manifest.py, in case people want to use that module for other purposes.